ptn/boozebot/botcommands/Departures.py

```diff
@@ -294,26 +307,56 @@ def sanitize_input(text):
         if departure_timestamp:
             departure_time_text = f" <t:{departure_timestamp}:f> (<t:{departure_timestamp}:R>) |"
         else:
-            departure_time_text = ""
-
+            departure_time_text = f" {bot.get_emoji(get_thoon_emoji_id())} |"
+        
         # Check if the departure needs a hitchhiker ping
-        if departure_location in ["N1", "N2"] and arrival_location in ["N1", "N0", "N0 Star", "N0 Planet 1", "N0 Planet 2", "N0 Planet 3", "N0 Planet 4", "N0 Planet 5", "N0 Planet 6"]:
+        hitchhiker_systems = [0, 1, 2]
+        
+        try:
+            departure_system_index = int(departure_location.split(" ")[0][1:])
+        except ValueError:
+            departure_system_index = 16
+
+        try:
+            arrival_system_index = int(arrival_location.split(" ")[0][1:])
+        except ValueError:
+            arrival_system_index = 16
+        
+        if departure_system_index in hitchhiker_systems and arrival_system_index in hitchhiker_systems:
             print("Departure needs hitchhiker ping.")
             hitchhiker_ping_text = f"| <@&{str(server_hitchhiker_role_id())}>"
             departure_time_text = f" {bot.get_emoji(get_thoon_emoji_id())} |"
         else:
             hitchhiker_ping_text = ""
-
+            
+        # Set the direction arrow text
+        if departure_system_index == arrival_system_index:
+            print("Departure and arrival are the same system.")
+            await interaction.edit_original_response(content="Departure and arrival systems cannot be the same.")
+            await steve_says_channel.send(f"Error for {interaction.user.name} during `/wine_carrier_departure` command: Departure and arrival systems cannot be the same.")
```

Comment:
```suggestion
            error_msg = "Departure and arrival systems cannot be the same."
            print(error_msg)
            await interaction.edit_original_response(content=error_msg)
            await steve_says_channel.send(f"Error for {interaction.user.name} during `/wine_carrier_departure` command: {error_msg}")
```

suggestion(non-blocking): slightly alters the console msg, but consistency is nice